packages/seacas/libraries/ioss/src/init/CMakeLists.txt

```diff
@@ -39,9 +39,9 @@ TRIBITS_ADD_LIBRARY(
 	Ionit
 	HEADERS	${HEADERS}
 	SOURCES ${SOURCES}
-	DEPLIBS Ioss ${EXODUSLIB}
-	        Iogn Iogs Iohb Iotr ${IOVISLIB}
-            ${PAMGENLIB} ${CGNSLIB} ${VISLIB} ${DATAWAREHOUSELIB} ${ADIOS2LIB}
+	DEPLIBS Ioss ${EXODUSLIB} ${PAREXOLIB}
```

Comment:
There is no need for `PAREXOLIB` or `EXODUSFAC` anymore.  There is a single `EXODUSLIB`

Reply:
Fixed in 778cc92.